system-int-test/test/cmr/system_int_test/ingest/collection_misc_validation_test.clj

```diff
@@ -1,26 +1,26 @@
 (ns cmr.system-int-test.ingest.collection-misc-validation-test
   "CMR Ingest miscellaneous validation integration tests"
   (:require
+    [clojure.java.io :as io]
     [clojure.test :refer :all]
-    [cmr.system-int-test.utils.ingest-util :as ingest]
-    [cmr.system-int-test.data2.collection :as dc]
-    [cmr.system-int-test.data2.core :as d]
     [cmr.common-app.test.side-api :as side]
     [cmr.ingest.config :as icfg]
-    [clojure.java.io :as io]))
+    [cmr.system-int-test.data2.core :as d]
+    [cmr.system-int-test.data2.umm-spec-collection :as data-umm-c]
+    [cmr.system-int-test.utils.ingest-util :as ingest]))
 
 (defn assert-valid
   ([coll-attributes]
    (assert-valid coll-attributes nil))
   ([coll-attributes options]
-   (let [collection (assoc (dc/collection coll-attributes) :native-id (:native-id coll-attributes))
+   (let [collection (assoc (data-umm-c/collection coll-attributes) :native-id (:native-id coll-attributes))
```

Comment:
Oh good - business rule validations probably cover a good chunk of the acceptance criteria. What's left to verify beyond those?